backend/docker-compose.yml

```diff
@@ -9,4 +9,8 @@ services:
     environment:
       MONGO_INITDB_ROOT_USERNAME: Kueens-b
       MONGO_INITDB_ROOT_PASSWORD: Kueens111
+      MONGO_INITDB_DATABASE: mydb
```

Comment:
I suggest renaming the database name to a meaningful name such as "Attendance" or something similar